src/static/css/main.less

```diff
@@ -4,16 +4,16 @@
    ========================================================================== */
 
 // import Normalize before all other LESS/CSS
-@import "../../vendor/normalize-css/normalize.css";
+@import (less) "../../vendor/normalize-css/normalize.css";
```

Comment:
That'd be great if you can find it @Scotchester, do you recall if it was just to include the (keyword) or to have everything `(less)`? [This'll treat `normalize.css` as a Less file](http://lesscss.org/features/#import-options-less), which doesn't seem like it would do any harm, but is also running things through the Less pre-processor that don't need to be.